src/main/resources/application.yml

```diff
@@ -43,3 +43,6 @@ sqs:
   queue.name: offender_events_ui_queue
   dlq.name: offender_events_ui_dlq
   endpoint.region: ${aws.region}
+
+ui:
+  pageSize: 100
```

Comment:
so by default show the last 100 records? Just wondering if the default should be much higher, just thinking about those BALANCE events we saw.

Reply:
Can be controlled by deployment config I think but just picked a random number to test with.  When it comes to adding features being able to include/exclude a message type will help, but we could also let the user determine the page size.